readme.md

```diff
@@ -64,7 +68,13 @@ Also, you can try [online version of this bot](https://t.me/gotelegramuidemobot)
 
 Small helper for easy building of inline keyboard.
 
-[Documentation](keyboard/inline/readme.md)
+![paginator.png](paginator/paginator.png)
```

Comment:
link to inline documentation change to paginator image?

Reply:
My bad, I'm going to change the link to the reply keyboard